lua/wikis/commons/Infobox/Person.lua

```diff
@@ -451,6 +547,60 @@ function Person:displayLocations()
 	end)
 end
 
+---@param roleKey string
+---@return PersonRoleData?
+function Person:createRoleData(roleKey)
+	if String.isEmpty(roleKey) then return nil end
+
+	local key = roleKey:lower()
+	local roleData = ROLES[key]
+
+	if not roleData and key ~= '' then
```

Comment:
```suggestion
	if not roleData then
```
isn't this redundant (due to the return nil on empty key check above)?

could also use early return ala
```lua
	if roleData then return roleData end

	local display = String.upperCaseFirst(roleKey)
	return {
		display = display,
		category = display .. "s"
	}
```